nncf/torch/quantization/quantize.py

```diff
@@ -148,13 +151,15 @@ def quantize_impl(model: torch.nn.Module,
                   subset_size: int,
                   fast_bias_correction: bool,
                   model_type: Optional[ModelType] = None,
-                  ignored_scope: Optional[IgnoredScope] = None) -> torch.nn.Module:
+                  ignored_scope: Optional[IgnoredScope] = None) -> NNCFNetwork:
     """
     Implementation of the `quantize()` method for the PyTorch backend.
     """
     if fast_bias_correction is False:
         raise ValueError(f'fast_bias_correction={fast_bias_correction} is not '
                           'supported')
+    nncf_logger.warning('Bias correction and fast bias correction algorithms'
```

Comment:
Please revert these changes. Torch supports default behavior and raise error if fast_bias_correction is False. This warning just generate noise.

Reply:
Agree

Reply:
Done